server/text_generation_server/models/causal_lm.py

```diff
@@ -994,14 +997,18 @@ def generate_token(
         else:
             token_idx = torch.tensor(batch.attention_mask.shape[-1] - batch.right_padding).to(self.device)
             input_ids = torch.index_select(batch.input_ids, 1, token_idx - 1)
-            batch.logits = self.forward(
-                input_ids,
-                batch.attention_mask,
-                batch.position_ids,
-                token_idx,
-                batch.past_key_values,
-                bypass_hpu_graph=prefill and self.limit_hpu_graph if self.enable_hpu_graph else None,
-            )
+            logits = self.forward(
+                    input_ids,
+                    batch.attention_mask,
+                    batch.position_ids,
+                    token_idx,
+                    batch.past_key_values,
+                    bypass_hpu_graph=prefill and self.limit_hpu_graph if self.enable_hpu_graph else None,
+                )
```

Comment:
just fixed indentation
```suggestion
            logits = self.forward(
                input_ids,
                batch.attention_mask,
                batch.position_ids,
                token_idx,
                batch.past_key_values,
                bypass_hpu_graph=prefill and self.limit_hpu_graph if self.enable_hpu_graph else None,
            )
```